activemodel/lib/active_model/dirty.rb

```diff
@@ -140,6 +140,13 @@ def initialize_dup(other) # :nodoc:
       @mutations_from_database = nil
     end
 
+    # Exclude +mutations_from_database+ attribute from json as it lead to
+    # recursive call to as_json method
```

Comment:
```suggestion
```

We don't add comments to code, only documentation. This is already covered by the commit message.